automation_tools/__init__.py

```diff
@@ -3115,13 +3116,39 @@ def download_manifest(url=None, consumer=None):
                          ' session and distributor hash')
 
 
+def assert_downloading_manifest():
+    """Makes sure that downloaded manifest matches manifest name from a reference file."""
+
+    user = os.environ.get('RHN_USERNAME')
+    password = os.environ.get('RHN_PASSWORD')
+    consumer = os.environ.get('CONSUMER')
+    product_names_file = os.environ.get('PRODUCT_NAMES')
+
+    r = requests.get('https://subscription.rhn.redhat.com/subscription/'
+                     'consumers/{0}/entitlements'.format(consumer),
+                     auth=(user, password), verify=False)
```

Comment:
How about adding a
`r.raise_for_status()` line here, so the script raises a proper HTTP exception in case the server does not return 200?